spec/features/user_views_reviews_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+# Acceptance criteria
+# [√] I must be logged in
+# [√] I must be on the bathrooms details page
+# [√] I can see a list of reviews ordered by most recent review
+
+feature 'As a user
+  I want to view all reviews on a bathroom
+  So that I can decide which bathroom to visit' do
+  scenario 'user visits bathroom details page
+    and sees reviews for that bathroom' do
+    @user = FactoryGirl.create(:user)
+    @bathroom = Bathroom.create!(
+      user: @user,
+      rating_average: 5.0,
+      location_name: "Starbucks",
+      street_address: "62 Boylston",
+      zip_code: "02116",
+      description: "Back corner. Key from barista. Watch out for joe."
+      )
+    @review = Review.create!(
+      user_id: @user.id,
+      bathroom_id: @bathroom.id,
+      body: "Favorite bathroom evahhhhhh",
+      rating: 3)
+    visit new_user_session_path
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: @user.password
+    click_button "Log in"
+    expect(page).to have_content "Signed in successfully"
+    click_link 'Check out bathrooms'
+    expect(page).to have_content "Starbucks"
+    click_link @bathroom.location_name
+    expect(page).to have_content 'Reviews'
+    expect(page).to have_content(@bathroom.location_name)
+    expect(page).to have_content(@review.body)
+    expect(page).to have_content(@review.rating)
+    expect(page).to have_content(@review.created_at)
```

Comment:
This test is kind of gigantic and hard to read! A couple things:

- You don't need `expect` statements for things that you're testing elsewhere, like a user getting successfully signed in and viewing a bathroom name on a list of bathrooms (lines 31 and 34). Remove both those expect statements so the test is more to-the-point!
- Add some whitespace in to separate the sections of your test. Maybe a new line before 27, and at 31 and 34